etc/docs/template/layouts/partials/footer.html

```diff
@@ -0,0 +1,8 @@
+<div class="footer">
+  <div class="copyright">
+    <p>Copyright © 2015 <a class="smalltext" href="http://www.mongodb.com">MongoDB, Inc</a>.
```

Comment:
I think we still hold the copyright :) so this could be 2022 (or maybe we don't need this?)